src/main/java/uk/ac/ebi/embl/api/validation/submission/SubmissionValidator.java

```diff
@@ -147,6 +147,10 @@ SubmissionOptions mapManifestToSubmissionOptions(Manifest manifest) throws Valid
             options.submissionFiles = Optional.of(setSequenceOptions((SequenceManifest) manifest));
         }
         options.assemblyInfoEntry = Optional.of(assemblyInfo);
+        if(manifest.getAuthToken()!=null) {
+            options.authToken = Optional.of(manifest.getAuthToken());
+        }
+        options.isDevMode = manifest.getTestMode();
```

Comment:
- It is not safe to think that isDevMode would have something to do with WebinCLi test mode, example:
```
if(!isWebinCLI || isDevMode) {
			if (!(new File(reportDir.get())).isDirectory())
				throw new ValidationEngineException("SubmissionOptions:invalid ReportDir");
```
- Instead we should add a new Option webinCliTestMode that is set to true if manifest.getTestMode() is true.